src/main/java/org/cloudfoundry/promregator/auth/OAuth2XSUAAEnricher.java

```diff
@@ -44,7 +41,7 @@
 		} else {
 			this.httpClient = HttpClientFactory.create(c.getClientIdentity());
 			this.tokenClient = new XsuaaTokenFlows(new DefaultOAuth2TokenService(this.httpClient),
-					new XsuaaDefaultEndpoints(c), c.getClientIdentity()).clientCredentialsTokenFlow();
+					new NullEndpointProvider(c), c.getClientIdentity()).clientCredentialsTokenFlow();
```

Comment:
Here we use our own endpoint provider which does not modify the urls. We do not add any path suffix.